packages-exp/app-check-exp/src/types.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { FirebaseApp } from '@firebase/app-exp';
+import { AppCheckToken } from './public-types';
+
+export interface FirebaseAppCheckInternal {
+  // Get the current AttestationToken. Attaches to the most recent in-flight request if one
```

Comment:
If these are public-facing comments, there are several places to consider: is it a literal or just a thing, like "attestation token."

Reply:
This file is internal only. Maybe we should come up with a comment header to indicate files where none of the comments are public?

Reply:
Public APIs/comments are marked with `@public`, so we can rely on it.